src/routes/about/+page.js

```diff
@@ -0,0 +1,352 @@
+import { tooltip } from 'svooltip';
+
+const dataSrc = `https://api.github.com/repos/spyder-ide/spyder/contributors?per_page=100`;
+
+const currentContributors = [
+  {
+    id: 365293,
+    name: "Carlos Córdoba",
+    role: "Lead mantainer",
+    tooltip: [
+      {
+        list: [
+           "Maintainer since 2013"
```

Comment:
```suggestion
           "Lead maintainer since 2013"
```

Just "maintainer" sounds like you were merely _a_ maintainer, rather than _the_ maintainer